### ISSUE TITLE
Test 4

### ISSUE THREAD

**Issue by josephdelon** [DrasilRNA/test#53]

**Comment by jilldevont** [comment]
I have gotten all the other statues, but the stupid statue in Kakariko Village has a, like, eyepatch thingy over where I need to use the Dominion Rod to be able to move the darn statue.

**Comment by jevoneorwells** [comment]
?od i od tahW .eltsac eluryh eht fo elddim eht ni ffo em detrats tI .eliforp ym no dekcilc i dna ,neercs niam eht ot kcab tnew ti neddus a fo lla nehT .gnihton ,etomer iiw eht dekcehc i .deneppah gnihton ,detiaw i ,dne eht diaS .neercs tsal eht ot tnew ti dna emag eht taeb I

**Comment by jevoneorwells** [comment]
?moor hctilg nonac eht gniretne retfa tiuq dna emag ruoy evas ot efas ti si nehW

**Comment by jevoneorwells** [comment]
sknahT?bew siht fo tuo teg ot od I od tahW ?ni ruoy moor eht ni retcarahc eht teser lliw taht obmoc nottub a ereht si ,nwod dna pu gnipmuj ereht sdnats tsuj eh ,nretnal eht esu ro evom em tel ton lliw ti won dna bew redips eht no deppets I ,yeknom a evas dna eloh eht nwod llaf ,bew eht nrub neht redips a llik ni klaw uoy erehw elpmet tserof eht ni mI

**Comment by jevoneorwells** [comment]
When you enter death mountain, a goron will block the path and I think the game expects me to throw him out of the way. The problem is, I could not even grab him. The grab button will just let the goron roll straight trough me. Please help.

**Comment by jevoneorwells** [comment]
.detaicerppa yllufetarg eb dluow pleh ynA.ti htaeneb gub eht esaeler ot roolf eht no xob eht hsup t'nac I ,yllacificepS .sneppah gnihton tub ,gnihsup eb ot sraeppa flow eht )A( hsup I nehw ,revewoh ,'hsuP' yas ot segnahc nottub )A( eht ,dehsup eb nac taht meti na ot txen ma I nehW

**Comment by jephrynorman** [comment]
?esle gnihtemos od ot evah I kniht I dna eid reve t'nseod eh tuB .llab a otni pu sllor eh dna hcamots eht ni mih tih uoy dna no stoob yvaeh ruoy evah ot evah uoy yug eht si eH

**Comment by jephrynorman** [comment]
?pu s'tahw wonk enoyna seoD .raf taht tog I ecnis revo yaw eht lla trats ot evah ot tnaw t'nod yllaer I .reh dnif t'nac I yhw tuo erugif ot gniyrt emit gniyalp fo tol a etiuq pu dekcar ev'I .erehwon ,ti edistuo ton ,esuoh reh ni toN dnuof eb ot erehwon s'ehs ,doR noinimoD daeD eht zapmI wohs ot nwot neddih eht ot kcab edir I nehw ,dna hcae eciwt tsael ta ot syas ti taht enoyreve htiw deklat ev'I .llaC esroH eht teg uoy erehw trap eht tsap nettog ev'I ,ssecnirP thgiliwT nI

**Comment by jevoneorwells** [comment]
I am trying to activate the howling stone in the secret village and it says I am supposed to pause during howling...how do i do that. If I try to pause, it starts over? What am i missing?Thanks for any help you can give me!

**Comment by johnasduval** [comment]
?pohsotohp ni krow revelc a ro ,gge retsae na taht si.ekaf a si ti gnimussa ,stohsneercs morf segami htiw edam saw ti ekil skool tI .gnissik aili dna knil fo erutcip gnikool gnicnivnoc yrev a ssorca emac dna enilno ffuts adlez pu gnikool saw I tub ,)emag eht netaeb evi ecnis yllaicepse( noitseuq diputs a si siht wonk I

**Comment by jephrynorman** [comment]
[: !PLEH .ti ni eloh eguh a sah egdirb nrad eht dna ,uryanaL ni krow dehsinifnu ,erus m'I ,htiw ,egdirb eht fo edis rehto eht no ma I ereh ,teY .flow eht sa tnecs taht wollof I erehw enoz thgiliwt eht deretne ydaerla ev'I dnA .sbmob evah ydaerla I ,lleW .okirakaK ni sbmob yub og dluohs I tniop siht ta taht stseggus dehctaw ev'I oediv yreve ,daer ev'I gnihtyrevE"!tegrof t'nod os ,oot ereht eb dluohs wodahs desuF ehT .egalliV okirakaK fo htron nellaf sah taht thgiliwt eht ot daeH .uoy rof doog ,lleW ?thgir ,ecnivorP uryanaL eht ni si rof gnikool er'uoy nosrep ehT" syas tsuj andiM .egdirB nidlE eht fo edis dleiF eluryH eht no m'I tub ,uryanaL ni eb ot em seriuqer ta emag eht tfel I erus m'I trap eht ,ffo tfel I erehw rebmemer nac I fi ees ot shguorhtklaw daer I nehw dna ,shtnom wef a ni emag eht deyalp t'nevah I

**Comment by zackeryhanice** [comment]
I need to get the Gale Boomerang but I don't know where or how to get into the room where he is in.

**Comment by jephrynorman** [comment]
\/evobA

**Comment by zackeryhanice** [comment]
?esoprup tuohtiw meht evom ot em wolla odnetniN dluow yhW ?ereh gnihtemos gnissim I mA .gnihton ....laever ot smees meht gnivoM .niaga meht evom tonnac uoy ,devom era yeht ecnO .evac eht ni somaeb htob rof eurt si sihT .roolf eht ni seloh evif laever dna meht evom ot elba m'I ,meht gnitaefed retfA .somaeb era ereht ailyH ekaL ni nrevac eht nI

**Comment by johnasduval** [comment]
em pleh esaelp os taht ekilemos ro ekans retaw a otni nrut ti wonk dna elcatnet eht fo tuo eye eht dellup ydaerla I .ti taeb ot woh wonk tnod i dna rieht ssob tsak eht gnithgif mi dna elpmeT debekaL ni m'I

**Comment by zackeryhanice** [comment]
It would be nice to be recognized as a hero and stuff.

**Comment by zackeryhanice** [comment]
I am stuck trying to help the zora kid and I don t have any bombs left to defeat the big goblin in twilight princess.PLEASE HELP

**Comment by jephrynorman** [comment]
?od I od tahW?od I od tahw.gnihtemos em llet ot sah ehs syas ehs esuac hguorht og t'nac I ,rood eht nepo ot yrt I nehw tuB.ereht t'nsi ,kcis s'ehs em sllet taht efiw s'iteY eht ,sniuR kaepwonS nI tuB.tce ,spam ,syek ,smeti lla htiw gnihtyreve gnirolpxe tsuj saw I nuf rof os ,)yalper noitca(RA na em tog dneirf ym dna hguorhtyalp ht5 ym si siht ,thgirlA

**Comment by jevoneorwells** [comment]
?sQAF ynam os ereht r yhw hsog tub si ti drah woh wonk t'nod I os emag siht deyalp reven ev'I

**Comment by johnasduval** [comment]
?yrotnevni ym ni ti dnif tonnac I pu ti kcip I retfa tuB .gof eht ssorca em teg ot ti desu yeknom eht esuaceb ytpme sti dna ,nretnal eht pu kcip I

**Comment by jevoneorwells** [comment]
._. .noitseuq diputs a sti wonk I ?nwot eltsaC eht ot og dna tuo og i nac woh ,thgiliwT eht morf ailyH ekaL gnieerf dna thgiL fo sraeT eht gnitcelloc retfA

**Comment by johnasduval** [comment]
?pleh esaelp u nac os .nwot eltsac fo eltsub eht ni ti dnif lliw i taht si yas dluow ehs lla .pleh tnow andim dna nwot eltsac ni erehwyreve dekool I

**Comment by jevoneorwells** [comment]
...taht retfa od ot tahw wonk tnod I tub yeknom rehtona devas dna srood lla werht tnew I dneirf reh devas i neht dna yeknom lrig eht devas I

**Comment by jennywaiver** [comment]
After meating the big fairy i went to the next room and there was a was blocking the next door how to u get past the wall?

**Comment by jephrynorman** [comment]
.seno rehto eht t teg tnac i tub meht fo wef a evah I

**Comment by jennywaiver** [comment]
Its not letting me put the master sword into the pedestal to make the stairs appear so i can enter the temple of time

**Comment by zackeryhanice** [comment]
After clearing the Goron Mine, and you get the ability to go to Lanayru Province by crossing the giant bridge(the one with the epic horseback duel to save Colin) and you come across some boulders and blow them up, three Shadow Beasts appear and a chunk of that same bridge is removed. Does anyone know of a way to restore that missing piece of the bridge?

**Comment by jevoneorwells** [comment]
After sky temple i have 3 piece miror and fix it in desert part then i come in door and start new part how can i beat black hand when take sun(ball) from black hand...

**Comment by jennywaiver** [comment]
)slaedro fo evac eht ta mi(?moor emas eht ni era taht stunkrad 3 taefed ot yaw tseb eht si tahW

**Comment by jephrynorman** [comment]
eht erehw tuoba desufnoc m'I woN

**Comment by johnasduval** [comment]
Please tell me where i can get the lantern .

**Comment by jephrynorman** [comment]
.em ekil tsuj eb ot ,niloC ,nos reh stnaw ehs ehs ehs ,reh ot klat I emit yrevE .yrotnevni ym ni ti ees t'nod I dna taht did I .teksab eht reh evig uoy retfa uoy ot ti sevig )eman reh tegrof I( revir eht yb namow eht dias ti ,rewsna eht pu dekool I nehW

**Comment by johnasduval** [comment]
Wen im in the graveyard and isense by the tree and the ghosts are pointing at the rock in front of the tree, wat do i do with it?

**Comment by jevoneorwells** [comment]
...aval eht revo pmuj tnac I

**Comment by jevoneorwells** [comment]
I need some money to get the magic armor, so I can get closer to 100%.

**Comment by jephrynorman** [comment]
I beat the game and it went to the last screen. Said the end, i waited, nothing happened. i checked the wii remote, nothing. Then all of a sudden it went back to the main screen, and i clicked on my profile. It started me off in the middle of the hyrule castle. What do i do?

**Comment by jennywaiver** [comment]
Where do I find the Cave of Ordeals and where are the fairy locations?

**Comment by jennywaiver** [comment]
How do I give the Goron the water so he can dig the tunnel from the south side of Castle Town?

**Comment by jephrynorman** [comment]
?kool I od erehW .niatnuom htaed ni reirrab eht dniheb tsaeb wodahs tsrif eht teg ot yaw a dnif ot deen I

**Comment by johnasduval** [comment]
I can't beat the floating hand in Zant's palace that guards the first Sol I take three lives and it stops glow I wait and it starts glowing again.And if I leave the room it takes the Sol back like I wasn't even there.Is there something I need to do? Please help me.

**Comment by jennywaiver** [comment]
)yllaretil( em gnillik era sdrib gnad eseht dna semit 91 ekil deid ev'I ?EREHT REVO TEG I OD WOH...si ti fI .si rewot tsellat eht erehw si siht fi em llet oS .rewot tsellat eht eht rewot eno siht morf pmuj ot gniyrt gniyd peek i .eno tsellat eht dnif ot gniyrt srewot eht fo pot no m'i woN .hE !sgniht kcalb elttil esoht lla htiw rewes ssorg taht fo tuo nettog ton tsuj evah I yakO

**Comment by jephrynorman** [comment]
.hcum pleh t'ndid ti tub qaF gnihsiF eht ta dekool evah I .ti tuohtiw ti htcac ot woh em llet ot enoemos deen I os tiab gorf eht rof laoG lloR taeb t'nac I

**Comment by jephrynorman** [comment]
?setarbetrev eht gniyortsed ot kcirt a ereht sI ?gnorw gniod I ma tahW .deneppah sah gnihton dna semit 001 enobkcab sih otni nur ot rennips eht desu evah I

**Comment by johnasduval** [comment]
Ok, I'm in the forest temple and I just beat the big plant thing and rescued the third monkey. I went to the main room again, and decided to go to a diffrent room. The monkey fell off the main platform thing, and I went into the other room with only two monkeys. Surprised, I went back into the main room to see the monkey that fell down now dancing on top of a pillar. I have no idea why he is doing this, but I know I need him in order to progress the temple. Any suggestions? It didn't say a thing about this in the guide I was following..

**Comment by johnasduval** [comment]
I can see the hole in the wall but can't figure out how to get the water to go down with out going back over the spikes. And I can't figure out how to make wolf link dive under the water.

**Comment by johnasduval** [comment]
im using dolphin and im at the point where i have to catch a reekfish but i cant reel it in as in no buttons work but A and the C stick 
and i used this controller when i had to get the fish for the cat and the controls worked fine any advice plz.

**Comment by jephrynorman** [comment]
I am four poe souls short of completing the side quest for Jovani. I know I need the one on level 44 of the Cave of Ordeals, and the other three are in Lanayru Province, according to the map data (which shows I have 16/19 there). I have looked at lists and videos for the souls' locations, but they are all jumbled together with no regional breakdown, so I somehow keep overlooking the ones I need, and the only video I've found that does list them by province says that Lanayru only has 10. Can anyone point me towards a good, descriptive list of poe souls specifically for the Lanayru Province? Or, better yet, a video?

**Comment by jevoneorwells** [comment]
tniop siht ta semit htob deppots dna emag siht urht emit dnoceS .elpmet debekal rof urht klaw dewollof ev'I .raeppa ton seod daot oked fi tohswalC teg ot emag ni ssergorp t'nac I

**Comment by jennywaiver** [comment]
?ssapmoc a neve ereht sI .orocooco dnuof i tub ,ereht ssapmoc a dnif t'ndid i esuaceb suoicipsus leef i tub elpmet noroG eht ni raf ytterp neeb e'vI

**Comment by johnasduval** [comment]
?emag ym gnitratser sediseb od nac I gnihtyna ereht sI .rehtruf yna og t'nac I os dekcol llits si rood dekcol eht tub ,niaga ti barg t'nac I os nepo si tsehc ehT .yek llams eht evah t'ndid I ,syeknom rehto eht eerf ot moor taht ot kcab emac I nehw dna ,gnaremoob elag eht gnitteg retfa tniop emos ta tiuq dna devas I os ,etal tog tI .elpmet eht ni ssergorp ot smoor rehto eht ot kcab tnew I os ,gnaremoob elag eht evah t'ndid I .rood eht yb thgir saw taht tsehc eht morf yek llams eht debbarg dna ,smoorhsum no gnipmuj era syeknom eht erehw ot uoy sdael dna egdirb derewop-gnaremoob elag a sah taht moor eht ot tnew I ,reilraE .elpmet tserof eht no gnikrow yltnerruc m'I dna ,retupmoc ym no rotalume CG a no ssecnirP thgiliwT hguorht gniyalp m'I

**Comment by jennywaiver** [comment]
What's the point of games when there are so many quests that you have to solve before you can say "I finished the game!"?

**Comment by johnasduval** [comment]
.sknahT .revo level taht trats ot yaw yna ro yek a tuohtiw yawllah llabnonnac 3 eht otni teg ot yaw rehtona ro teg nac I yek rehtona ereht sI .tniop gnivas rehto ylno ym ta ylsuoiverp sruoh 52 emag eht tratser ot tnaw t'nod dna yek rehtona gnitteg fo yaw on ees I .tsaeb eci eht pu wolb dna revel eht no sllab nonnac eht tup ot rood eht fo daetsni kaepwons ni klaw uoy sa edis dnah thgir eht no rood eht denepo I

**Comment by jephrynorman** [comment]
In lake hylia

**Comment by jevoneorwells** [comment]
.sknahT .em pleh esaelP .sseccus on htiw tub ,tegrat L tuohtiw ro htiw ti gnisserp ,yldetaeper A gnisserp ,ecno A gnisserp sa hcus syaw rehto ynam deirt I .sdrawretfa dnah reh esiar ton seod ehs tub ,yhw wonk t'nod I ,emit tsrif eht rof dnah reh sesiar ylno andiM ,hguohtlA .ereh krow t'nseod yllaer ti tub ,meht pots ot woh wonk ydaerla I dna snoroG dna staog gnippots ekil si ti wonk I .toof sih rednu mih selpmart nonaG esuaceb truh steg kniL floW taht tpecxe ,sneppah gnihton ,nottub A eht dloh dna sserp I nehw tub ,)?lamron taht si ,dnoces a naht ssel ebyam ,ytrohs yrev hguohtla( neercs eht no raeppa lliw nottub A sserp ot tpmorp eht dna dnah reh sesiar andiM .mih taefed ot redro ni kniL floW otni nrut dluohs I wonk I ,drows dna wob htiw eciwt nonaG gnitaeb retfA

**Comment by jennywaiver** [comment]
I've noticed that I have finished this game twice, and in the process of trying to finish it a third time. When I am not playing, I feel sad and tired. When I play the game, I feel happy and excited. Every day, I can't wait to get home to play this game because it makes me feel better about myself. All I think about, is this game. Do I have an addiction? Or do I have a different problem in my life that makes me feel emotionally attached to this game?

**Comment by jevoneorwells** [comment]
!sknahT ,detaicerppa si pleh ynA...ti dnuora tog yeht woh/fi dna emag rieht htiw eussI siht dah sah esle enoyna fi gnirednow saw I oS...nees ev'I taht skrow gnihton dna ,ni kcab dna moor eht fo tuo nar neve ev'I ,mrof flow otni gnignahc ot ,stooB yvaeH eht ffo dna no gnikat ot ,andiM ot gniklat morf gnihtyreve deirt ev'I...dnuorg raluger eht no s'eh nehw neve ,dnas eht ni kcuts s'eh ekil tca lliw kniL dna nwod wols ot trats ylmodnar ll'i ereht ni m'I revenehW ,syawynA...cte cte hguorht teg ot dnuora nur neht dna kcab llup ot evah uoy taht llaw a hcaer uoy litnu htap niatrec a wollof uoy ekam taht sekips eht htiw moor egral eht ni er'uoy nehw sdnuorG s'retibrA eht ni hctilg a eb ot smees tahw ssorca nur ev'I dna emag siht fo noisrev iiW eht gniyalp yltnerruc m'I ,olleH

**Comment by jennywaiver** [comment]
?od i od tahW .eltsac eluryh eht fo elddim eht ni ffo em detrats tI .eliforp ym no dekcilc i dna ,neercs niam eht ot kcab tnew ti neddus a fo lla nehT .gnihton ,etomer iiw eht dekcehc i .deneppah gnihton ,detiaw i ,dne eht diaS .neercs tsal eht ot tnew ti dna emag eht taeb I

**Comment by jennywaiver** [comment]
I've been trying to collect all Poe Souls but it gets kinda tiresome to wait till dark all the time.

**Comment by jevoneorwells** [comment]
I have the compass, big key, but the main hub room is only filled with water to level 2, not to the boss room. Can't think what I missed... ???

**Comment by johnasduval** [comment]
od ot tahw aedi on evah I won dna emag eht taeb I retfa niaga hguorht deyalp I esuaceb kcoltfos siht xif I od woH

**Comment by jephrynorman** [comment]
?sgorf hctac uoy od woH

**Comment by jennywaiver** [comment]
!elpoep ereh tuo em pleh esaelp oS !etag REHTONA si ereht sgniht evac ELTTIL esoht fo eno ni og I emit yreve esuac dam em gnikam sti dnA !erehw aedi on evah I tuB ,ereht dnuora erehwemos yek a si ereht taht gnileef a evah I .dnuora gninnur sgniht kcalb fo hcnub a htiw evac a ekil ni m'I dnA .ti dnuora teg ot woh tuo erugif ot dah I em dlot dna etag a dniheb em dekcol eh neht .ffuts dna tuo em gnipleh saw yuG elttil sihT dna...emit tsrif eht rof flow a emoceb tsuj evah I erehw ecalp eht ta mI os...seman gnirebmemer ta doog ton ma I esauceb ,ereh etarucca yrev eb ot gniog ton mi os yakO

**Comment by jennywaiver** [comment]
I have beatin the Desert temple but i can't get the Boss key can someone please help...?? thanks.... greatgame10

**Comment by jephrynorman** [comment]
It is black and gold and it's supposed to spin.

**Comment by jennywaiver** [comment]
ybabyrc oot uoy pleh smeti ym flah deirt i moor ssob eht ni mI

**Comment by jevoneorwells** [comment]
Was messing with the Options earlier and trying to find the perfect brightness setting, though in the end I figured I'd rather just stick to the default. Problem is I've forgotten what the default was. I want to say it's 3 pips out of 6 but I'm not sure. Sorry for the dumb Q. :I

**Comment by johnasduval** [comment]
Were can i find the boss key for the temple in the scared groove after you find the second statue?

**Comment by jevoneorwells** [comment]
Help! It's always day in Faron woods, even when its night everywhere else. Which means I can never get the poe. I've tried warping when its night, I've tried running around to make it night, but no matter what I try, I have not found success. And I've Googled it loads of times. Am I missing something? Because I really want to finish the hunt for poes.

**Comment by jevoneorwells** [comment]
Water spouts

**Comment by jennywaiver** [comment]
I should defeat the spiders on the vine in the forest temple but I'm a out of bullets, where I could find some bullets or go back to the village to take some bullets? Thank you

**Comment by johnasduval** [comment]
.dnuorg eht otno retaw eht gnipmud pu dne tsuj I llits dna ,taht gniod ton deirt ev'I ,mih otno gnikcol-L deirt ev'I ?mih ot retaw eht evig uoy od woh ,)htap eht gnikcolb redluob eht fo tnorf ni noroG eht( nwoT eltsaC fo htuos noroG eht ot ti reviled ot og dna )nwot eht pota morf ro seniM noroG eht morf rehtie( retaw gnirpS toH eht teg uoy nehW

**Comment by jephrynorman** [comment]
.tsaf og esroh sekam tsuj nottub A ehT .nottub poohw on evah .nrab otni staog teg t'naC

**Comment by johnasduval** [comment]
ezorf ti erofeb egami sal eht tsuj si neercs eht tub gninnur llits si emag eht rotalume eht esolc t'nseod ti ekil sezeerf emag eht amhogomrA fo thgif eht htiw rorre 1 evah i dna won yb elihw a rof noisrev ebuCemaG ssecnirP thgiliwT gniyalp neeb evah i oS

**Comment by jennywaiver** [comment]
Just curiosity

**Comment by jevoneorwells** [comment]
I'm in the main room in the Lakebed Temple with the moving stairs but I can't pull the switches or chains to move the stairway?

**Comment by jennywaiver** [comment]
ereh yas ot hcum toN .od ot evah I tseuq edis tsaL

**Comment by jennywaiver** [comment]
How do you get by the area after you meet the great fairy in Gerudo Desert? To get to the place you have to take the bridge piece that was destroyed and put back to connect Hyrule Field. The bridge piece goes over the Zora River. Then you warp back as a wolf and you can either stay a wolf or transform back into a human. Than you go down a stair case which was blocked by the bridge piece which was standing up. Then you fight a bunch of monsters in about 5 rooms. Then you meet he great fairy and she said she put fairies all over Ordon Spring. Then you go into the next room i killed a bunch of rates and these things that charge at you with metal on their back and the only way to kill them is to hit them in the butt three times. Finally the way to the next room is high up and i don't know how to get up there. I tried using my claw shot on all the places to use it on the roof, but none of them seem to be above the exit that's high up.

**Comment by johnasduval** [comment]
I have absolutely NO idea where to go after the desert prision.

**Comment by jennywaiver** [comment]
My horse is "hurt" and the mayor's daughter took it away from me. She went to the spring which is now blocked by a gate. Collin (who is also behind the gate) said to go through a tunnel that is accessable through the forest, which is also blocked by a gate. I don't know how to get through the forest gate. I think it has something to do with a man in town. He said he needed to sell something to the store so his wife wouldn't be mad. When I drank the larvae next to him he said it would be nice to have larvae. Maybe that has something to do with it but I don't know how to give it to him. How do I through the gates!?!?!

**Comment by johnasduval** [comment]
acsecnarFsreehC?melborp egnarts siht htiw pleh enoyna nac !yenom ruo fo etsaw a dna lla ta nuf on - sruoh demees tahw rof deirt i.....staog eht narrevo dna retsaf tnew ew tnaem tsuj hcihw 'hsad' ylno dluoc ew - A nottub no poohw'' eht gnissim erew ew taht derevocsid dna og rehtona dah i - ni yna teg t'ndluoc eh soc pu evag eh dna deyalp nos ym neht - gnidreh taog eht htiw melborp on dah i deyalp I emit tsrif ehT

**Comment by johnasduval** [comment]
How do i find the third suit? I have found the Zora armour and i have defeated Gandorf, but dont know where to get the third suit.

**Comment by jennywaiver** [comment]
I have horse back, and need to go learn the new skill in Ordon Village to beat the Goron on top of the mountian but I cant get past the rocks that are blocking the way what do I do?

**Comment by johnasduval** [comment]
?kcab reh teg I nac woh dna ,og ehs did erehW !enog s'anopE dna ,sniatnuom eht ot ot og ot desoppus m'I won erehw ,thgiliwt fo dnal eht fo trap dnoces eht tuo gniraelc dehsinif tsuj I

**Comment by johnasduval** [comment]
?setarbetrev eht gniyortsed ot kcirt a ereht sI ?gnorw gniod I ma tahW .deneppah sah gnihton dna semit 001 enobkcab sih otni nur ot rennips eht desu evah I

**Comment by jennywaiver** [comment]
I think i know where the sword is, but I cant get to it.... Please help.

**Comment by johnasduval** [comment]
About how long does it take to beat this game?

**Comment by johnasduval** [comment]
?elif emag eht teser I dluohS .tirips eht ot deklat t'nevah I .reviR s'aroZ reppU fo flah taht ot teg tonnac I ..tuB .enots gnilwoh eht raen ssarg eht ot og dluohs I taht smurof eseht no was I .noitseuq eht fo tuo si noitatropsnart rof srevir gnisu oS ,revo nezorf si gnihtyrevE .tropelet ailyH ekaL eht teg ton did I .mrof flow ni ma I

**Comment by johnasduval** [comment]
Where are they?

**Comment by jennywaiver** [comment]
)...eno tsesolc eht si siht snoitpo eht lla fo tuo tub ,noitseuq siht tif yllaer t'nseod "troppus hcet" wonk I ,sey(

 ?02$ artxe na tsael ta htrow yllaer emag siht si noinipo ruoy ni ,os fI ?scihparg eht tsuj snoisrev owt eht neewteb ecnereffid ylno eht sI .trapretnuoc redlo s'ti ot derapmoc evisnepxe adnik s'ti tub ,elihw a rof tsil tekcub ym no neeb sah emag sihT

**Comment by johnasduval** [comment]
I can't get sera there cat back to his owner in ordon village any pointers!

**Comment by johnasduval** [comment]
I was playing Twilight Princess for the wii and I finally came to the end of the twilight dungeon, I needed to get the boss key this is done by killing the falling enemies in the final room of the dungeon. As I went in there were 3 enemies which I killed both, but after that none spawned or fell.

Please help me.

**Comment by jennywaiver** [comment]
?redips gib eht evoba elpmeT tseroF eht ni llah niam eht ni tsehc eht teg i od woh

**Comment by johnasduval** [comment]
??lliks neddih tsal eht teg ot enots gnilwoh tsal eht dnif i od woH

**Comment by johnasduval** [comment]
!!!!ESOLC OS si ehs pleh sdeen ehS!syug esaelP?dnif ot DRAH YLLAER era taht era emos erehw wonk uoy fo yna od,yawynA...ygniht aroZ eht htiw ylno erus ylltcaxe ton m'i,tuB...egalliV okirakaK ni kniht I dna niamoD aroZ eht ni seno eht tog ydaerla ehs...yawa 81 s'ehs,lleW...derehtag era stac eht lla erehw nwoT eltsaC ni dnif uoy yug taht rof seop 06 gnidnif ot ESOLC os s'ehs....noitautis kcuts fo tib a etiuq ni s'retsis ym lleW?thgir seop fo draeh ev'uoy,syug yakO

**Comment by jennywaiver** [comment]
On the Box short, there is a picture of a magic meter, but there's no Magic meter in the game. Why did the Developers dump the Magic Meter?
